uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/App.java

```diff
@@ -76,6 +81,8 @@ public App(@NonNull final String packageName, @NonNull final String appName) {
     public App(@NonNull final String packageName, @NonNull final IAppInstaller appInstaller) {
         this.appInstaller = appInstaller;
         this.packageName = packageName;
+        // update installer is PlayStore by default
+        this.updateAppInstaller = new PlayStore();
```

Comment:
I think tis is fine for now, but it would actually make more sense for LocalApk installer to be the default here as usually we would be testing update scenarios with a side loaded APK

Reply:
Changed it to use LocalApkInstaller instead